diesel/src/query_dsl/mod.rs

```diff
@@ -265,7 +265,8 @@ pub trait QueryDsl: Sized {
     /// #         .first::<i32>(&connection)?;
     /// let join = users::table.left_join(posts::table);
     ///
-    /// // By default, all columns from both tables are selected
+    /// // By default, all columns from both tables are selected.
+    /// // Note, the structs must contain all the fields from original schemas in order.
```

Comment:
```suggestion
    /// // By default, all columns from both tables are selected.
    /// // For this case the loading target type (struct, enum) must contain all the fields from original schemas in order.
```

Would this be better, not sure if "loading target type" is a good terminology because I don't understand it on first glance.

Reply:
What's about
```suggestion
    /// // By default, all columns from both tables are selected.
    /// // If no explicit select clause is used this means that the result type of this query must contain all fields from the original schema in order.
```

Reply:
"result type of this query" does not seemed that good but I guess still okay, even though it is a bit confusing to me at first glance.